gen_cert.py

```diff
@@ -85,14 +85,18 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
-
+    utc_zone = get_timezone('UTC')
```

Comment:
We shouldn't hard-code this. Instead, we should pull it dynamically from `settings.TIMEZONE` (I think?).